portal-frontend/src/app/features/edit-application/land-use/land-use.component.html

```diff
@@ -0,0 +1,155 @@
+<section>
+  <div>
+    <h6>Describe the current land use of parcel(s) under application and adjacent parcels.</h6>
+    <app-warning-banner>
+      Please consult the 'What the Commission Considers' page of the ALC website for more information.
```

Comment:
This should be a link to https://www.alc.gov.bc.ca/application-and-notice-process/applications/what-the-commission-considers/